lib/openstax/salesforce/remote/lead.rb

```diff
@@ -16,7 +16,8 @@ class Lead < ActiveForce::SObject
     field :adoption_status,     from: "Adoption_Status__c"
     field :num_students,        from: "Number_of_Students__c"
     field :os_accounts_id,      from: "OS_Accounts_ID__c"
-    field :accounts_uuid,       from: "accounts_uuid__c"
+    field :accounts_uuid,       from: "accounts_uuid_c__c"
```

Comment:
`accounts_uuid_c__c` is how the field is described in SF.  Probably a typo, but we're rolling with it.